buildifier/utils/tempfile.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 )
 
+// Tempfile creates temporary files and cleans them up in the end
```

Comment:
nit: case Tempfile or TempFile -- probably the second and then be consistent.

And the doc here 'creates' reads like a func, not a struct

Reply:
Fixed